ethcontract-common/src/artifact/hardhat.rs

```diff
@@ -181,27 +181,27 @@ impl HardHatLoader {
         self
     }
 
-    /// Loads an artifact from a JSON value.
+    /// Load an artifact from a JSON value.
```

Comment:
In this particular case I actually prefer "Loads", and I think (but not completely sure) that "Loads" is actually more correct here. 

Example Scenario
- Object: Car
- Function: Drive on road
- Description: Drives car on road <---- Not "drive on road"

```py 
# drives self on road
def drive_on_road(self: Car):
```

Reply:
You're right, and I was using the wrong form this whole time XD I'll revert this change and update other comments.

Reply:
Wait, so we have a mix of both in our documentation comments. My inner perfectionist is not very happy about it =)